bin/deferred_work_manager

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require "tom_queue"
+
+module TomQueue
+  class Daemon
+    include LoggingHelper
+
+    def run(argv = ARGV)
+      return unless load_rails!(".")
+
+      deferred_manager = DeferredWorkManager.new
+      deferred_manager.start
+    end
+
+    # Load Rails environment
+    #
+    # Returns true/false indicate whether Rails is loaded or not
+    def load_rails!(path)
+      rails_path = File.expand_path(File.join(path, 'config/environment.rb'))
+      if File.exists?(rails_path)
+        info "Loading rails from `#{rails_path}`"
+
+        ENV['RACK_ENV'] ||= ENV['RAILS_ENV'] ||= 'development'
+        require rails_path
+        ::Rails.application.eager_load!
+        true
+      else
+        false
+      end
```

Comment:
> Currently if we call it from the wrong directory, then it exits silently (& successfully!).

ATM it will definitely fail if it's called from a wrong directory because in that case `default_prefix` and `bunny` is left un-configured. 

I'm actually thinking about starting the deferred_manager by passing a `amqp_config.yml` as an option. Good point though - handled in https://github.com/fac/tom_queue/commit/b084798d6773fcee27c83c8e502c19afd7bfd814